pages/resources/faq.md

```diff
@@ -24,9 +24,9 @@ Other networks are in earlier development and being actively worked on by the Po
 We intend to develop all five subprotocols into production-ready public networks, starting with the History Network. The State network is the next priority so that we can support light clients. Building out this core infrastructure is our primary goal. We believe many use cases will emerge when the infrastructure is in place and we anticipate working with Ethereum client developers and application developers wanting to develop truly decentralized backends for their products.
 
 
-## Why develop four clients in parallel?
+## Why develop mutiple clients in parallel?
```

Comment:
`multiple` not `mutiple` for all these changes

Reply:
should be good now